scss/_back-to-top.scss

```diff
@@ -24,7 +24,12 @@
   position: sticky;
   top: $back-to-top-link-offset-top;
   pointer-events: all;
-  @include button-icon($back-to-top-icon, $size: $back-to-top-icon-width $back-to-top-icon-height, $pseudo: "after");
+  @include button-icon(
```

Comment:
Looking back to this, I don't know if this should be in this PR, maybe open a brand new PR with all of your changes !

Reply:
Ok, but for the new PR I need a new issue ?